src/ContentScript/injectors/youtube.plugin.ts

```diff
@@ -98,6 +106,11 @@ class YoutubePlugin {
   }
 
   private insertLikeCoinButton(likerId: string) {
+    if (!likerId) {
+      const b = document.querySelector('.button-container');
```

Comment:
🤔This class name looks quite general...
Is it possible to change to something like `like-button-container` so it won't break Youtube functionality when there's change on Youtube site in the future?